dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -68,17 +69,20 @@ def is_of_supported(device_id=0):
         return is_of_supported_var
 
```

Comment:
idea, put this function in test_optical_flow.py and then just import it here.

Reply:
Done